<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>尘歌壶家具控制模块——派蒙模块</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/pm.png</normaloff>:/images/pm.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1000</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>600</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget{
	background-image: url(:/images/bg.png);
}</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>140</y>
       <width>489</width>
       <height>448</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="btn_curtain">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#btn_curtain
{
	background-image: url(:/images/curtain_on.png);
	background-color: transparent;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btn_door">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#btn_door
{
	background-image: url(:/images/door_off.png);
	background-color: transparent;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_airc">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#btn_airc
{
	background-image: url(:/images/airc_off.png);
	background-color: transparent;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btn_led">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#btn_led
{
	background-image: url(:/images/led_off.png);
	background-color: transparent;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btn_disconnect">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>30</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btn_disconnect
{
	background-image: url(:/images/connect.png);
	background-color: transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="info">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>80</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string> 连接失败，点击重新连接 </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="pasc_value">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>162</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift SemiLight</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>20</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="hum_value">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>242</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift SemiLight</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>15</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="tmp_value">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>324</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift SemiLight</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>30</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lux_value">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>406</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift SemiLight</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>15</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="wet_value">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>492</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift SemiLight</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>65</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="pasc_union">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>150</y>
       <width>100</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>70</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift SemiLight</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>kpa</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="hum_union">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>230</y>
       <width>100</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>70</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift SemiLight</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>m</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="tmp_union">
     <property name="geometry">
      <rect>
       <x>323</x>
       <y>315</y>
       <width>100</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>70</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift SemiLight</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>℃</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lux_union">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>398</y>
       <width>100</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>70</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift SemiLight</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>lux</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="wet_union">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>480</y>
       <width>100</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>70</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift SemiLight</family>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>%</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_mode">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#btn_mode{
	
	background-image: url(:/images/automode_on.png);
	background-color:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_setting">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#btn_setting{
	
	background-image: url(:/images/chilun.png);
	background-color:transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="maximumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
